Content.Server/Explosion/EntitySystems/TriggerSystem.cs

```diff
@@ -75,6 +77,7 @@ public sealed partial class TriggerSystem : EntitySystem
         [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
         [Dependency] private readonly SolutionContainerSystem _solutionContainerSystem = default!;
         [Dependency] private readonly InventorySystem _inventory = default!;
+        [Dependency] private readonly ElectrocutionSystem _electrocutionSystem = default!;
```

Comment:
```suggestion
        [Dependency] private readonly ElectrocutionSystem _electrocution = default!;
```
modern standard

Reply:
I don't think there is a standard for this?

Reply:
> I don't think there is a standard for this?

There is. Don't include system in the name